File: app/src/main/java/com/zawzaw/savethelibrary/model/gson/GsonNews.java
package com.zawzaw.savethelibrary.model.gson;

import com.google.gson.annotations.SerializedName;
import java.util.List;

/**
 * Created by zawzaw on 25/12/17.
 */

public class GsonNews
{
    @SerializedName("total")
    private int total;

    @SerializedName("data")
    private List<GsonNew> news;

    public int getTotal()
    {
        return total;
    }

    public List<GsonNew> getNews()
    {
        return news;
    }

}

File: app/src/main/java/com/zawzaw/savethelibrary/model/gson/GsonNews.kt
package com.zawzaw.savethelibrary.model.gson

import com.google.gson.annotations.SerializedName

/**
 * Created by zawzaw on 31/12/17.
 */

class GsonNews {

    @SerializedName("total")
    val total: Int = 0

    @SerializedName("data")
    val news: List<GsonNew>? = null

}
